### ISSUE TITLE
Call getBounds() on map using invokeSelf()

### ISSUE THREAD

**Issue by padmalcom** [xdev-software/vaadin-maps-leaflet-flow#282]
Hi, I try to get the boundaries of the currect view on the map. Leaflet implements a function called getBounds() that I like to call. What I do is:
```
PendingJavaScriptResult invokeSelf = map.invokeSelf(".getBounds()");
invokeSelf.then(j -> System.out.println("json:" + j.asString()));
```
But the result is always null. Do you have an idea why invokeSelf() does not work as I expect?

**Comment by AB-xdev** [comment]
Should be fixed in 4.0.1 (released now)

**Comment by padmalcom** [comment]
Thanks, I found a solution this morning using a ClientCallable.

```
String moveUpdateFunction = """
	function updateBoundingBox() {
		document.getElementById('%s').$server.updateBoundingBox(%s.getBounds());
	}
""";
		
map.on("moveend", String.format(moveUpdateFunction, ID, mapComponentJsAccessor));
...
@ClientCallable
private void updateBoundingBox(JsonObject json) {
	this.boundingBox = parseBoundingBoxJson(json.toJson());
}
```
